examples/batch_statistical.ipynb

```diff
@@ -0,0 +1,1030 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "example_input.plot()"
```

Comment:
By writing `example_input.plot();` you don't get that `<AxesSubplot:>` output shown.